projects/components/src/datagrid/interfaces/datagrid-column.interface.ts

```diff
@@ -234,6 +234,11 @@ export interface GridColumn<R> {
      * and sorting feature on. And we want filtering to be off on some columns while still having sorting enabled.
      */
     sortBy?: string;
+
+    /**
+     * When disableCliptext is true, the cells within the datagrid will not be clipped.
+     */
+    disableCliptext?: boolean;
```

Comment:
This should take options to be passed the the directive, one of them could be the option to disable it, as mentioned in a different comment.